### ISSUE TITLE
resume resets learning rate

### ISSUE THREAD

**Issue by barrh** [NervanaSystems/distiller#145]
Trying to train mobilenet, I had to pause and resume the training after roughly 100 epochs.
When I resumed the training, I used the same arguments (including lr) + --resume=checkpoint
When I resumed the training from checkpoint, the learning rate has been reset to its initial learning value.

Perhaps it behaves that way by design, but the expected behavior IMO would be that it resumes also the learning rate at epoch 100.

**Comment by barrh** [comment]
I found that apputils.load_checkpoint doesn't load the optimizer at all.
Fix is on the way.

**Comment by barrh** [comment]
duplicate of issue #70
